docs/mevblocker/builders/_category_.json

```
@@ -0,0 +1,6 @@
+{
+    "position": 2,
+    "label": "Builders",
+    "collapsible": true,
+    "collapsed": true
+}
```

```suggestion
{
    "position": 2,
    "label": "Builders",
    "collapsible": true,
    "collapsed": true
}

```
Add white space (new line)